EXOSIMS/util/version_util.py

```diff
@@ -0,0 +1,103 @@
+from importlib import metadata
+import platform
+import subprocess
+import os
+
+
+def get_git_info():
+    """
+    Get Git information including commit hash and status of changes
+    """
+    try:
+        # Check if we are in a Git repository
+        subprocess.run(
+            ["git", "rev-parse"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            check=True,
+        )
+
+        # Get the current commit hash
+        commit_hash = (
+            subprocess.check_output(["git", "rev-parse", "HEAD"])
+            .strip()
+            .decode("utf-8")
+        )
+
+        # Check for uncommitted changes
+        status = (
+            subprocess.check_output(["git", "status", "--porcelain"])
+            .strip()
+            .decode("utf-8")
+        )
+
+        uncommitted_changes = status != ""
+
+        return commit_hash, uncommitted_changes
+    except subprocess.CalledProcessError:
+        return None, False  # Not a Git repository
+
+
+def is_editable_installation():

```

Comment:
This does not appear to be a robust solution across platforms.  We need to figure out a different approach.